test/controllers/surveys_controller_test.rb

```diff
@@ -40,11 +62,15 @@ class SurveysControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should get new" do
+    sign_in @user
     get new_survey_url
     assert_response :success
   end
 
   test "should create survey" do
+    @survey = create(:survey, user: @user)
```

Comment:
think about this one...

Reply:
I see @spencerldixon I need to actually create the survey through a request with params rather than create it in the test.